README.md

```diff
@@ -105,12 +105,14 @@ When the Raspberry Pi boots up, give these answers to the following questions:
    ``` cd vocalfusion-avs-setup```
 
    And then either
-   
-   XVF3510: ```./auto_install.sh xvf3510```  
-   XVF3500: ```./auto_install.sh xvf3500```  
-   XVF3100: ```./auto_install.sh xvf3100```  
+
+   XVF3510: ```./auto_install.sh xvf3510```
+   XVF3500: ```./auto_install.sh xvf3500```
+   XVF3100: ```./auto_install.sh xvf3100```
    XVF3000: ```./auto_install.sh xvf3100```
 
+   If you need to use the keyword detection using the GPIO2 pin, use the flag '-g' after the device name. If this option is not selected, the Sensory engine is used.
```

Comment:
Does this option work for all products?

Reply:
yes, it can be used with all the product, but the only benefit I see is that you can compile the SDK without Sensory. The pin is used for KWD only in 3615.

Reply:
Okay, thanks for letting me know.